Src/Json/Shared.Json/ObjectDiffPatch.cs

```diff
@@ -0,0 +1,378 @@
+﻿#region *   License     *
+/*
+    SimpleHelpers - ObjectDiffPatch   
```

Comment:
JSON.NET doesn't contain anything like this OOB? Or isn't there a NuGet package for this code? I'm a bit wary that we have to maintain this now.